radar/platforms-and-services/aws-app-runner.md

```diff
@@ -0,0 +1,12 @@
+---
+title: "AWS App Runner"
+ring: adopt
+quadrant: "platforms-and-aoe-services"
+tags: [cloud, aws, serverless]
```

Comment:
Shouldn't the last tag be a value in `computing/storage/monitoring/networking`?

Reply:
Mmmm maybe, however I should use for this service compute like for an ec2 and it's weird. I don't know

Reply:
`Containerization`? Just to be clear, to me `serverless` is ok as long as we all agree on the possible tag values